commands/Eval.cs

```diff
@@ -36,6 +36,39 @@ public static void Vroom(IUser client, String args)
                     client.Vroom = vroom;
             }
         }
+        
+        public static void reports(IUser client)
+        {
+            if (list.Count == 0)

```

Comment:
See this: https://github.com/Nemo2478/sb0t-1/commit/eee0e3c072089d62d94938eb6757e0dee7c126e9